### PR TITLE
Tab bar visible, fix styles

### PR DESCRIPTION
**Pull request by DimaKelek**
1. Visible tab bar
2. added headers
3. fix alignment for create channel button

### CODE REVIEW

File: agoraStreaming/src/Components/Header/styles.ts [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
   container: ViewStyle;
   title: TextStyle;
+  titleContainer: ViewStyle;
   wrapperSectionIcons: ViewStyle;
   wrapperIcon: ViewStyle;
 };
 
 const SIZE_BLOCKS_ITEM = 35;
 
+const {width} = Dimensions.get('window');
```
Comment (Mentos15) [inline agoraStreaming/src/Components/Header/styles.ts:13]: use hook useDimentions in component (inline style)

File: agoraStreaming/src/Components/HorizontalCalendar/Helpers/scrollToSelectedDate.ts [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
+
   const monthMismatch = selectedMonth !== month;
   const yearMismatch = selectedYear !== year;
 
-  const oneDayWidth = width / VISIBLE_COUNT_DAYS;
-  const threeDaysScroll = oneDayWidth * Math.floor(VISIBLE_COUNT_DAYS / 2);
-  const scrollFromCurrentDay = selectedDayIndex
-    ? selectedDayIndex * oneDayWidth - threeDaysScroll
-    : 0;
+  const scrollFromCurrentDayHandler = (index: typeof selectedDayIndex) => {
```
Comment (koverchik) [inline agoraStreaming/src/Components/HorizontalCalendar/Helpers/scrollToSelectedDate.ts:17]: I think you need to change this type.

File: agoraStreaming/src/Redux/reducers/Live/index.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import {LiveActions, LiveActionsType} from '../../actions/LiveActions/types';
+import {LiveInitialStateType} from './types';
+
+const LiveInitialState: LiveInitialStateType = {
+  joined: false,
+};
+
+export const liveReducer = (
+  state = LiveInitialState,
```
Comment (Mentos15) [inline agoraStreaming/src/Redux/reducers/Live/index.ts:9]: why use UpperCase for first letter for const?

File: agoraStreaming/src/Redux/selectors/LiveSelectors/index.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import {RootState} from '../../store';
+
+export const selectJoined = (state: RootState) => state.live.joined;
```
Comment (Mentos15) [inline agoraStreaming/src/Redux/selectors/LiveSelectors/index.ts:3]: use name getIsJoined

Comment (Mentos15) [inline agoraStreaming/src/Redux/selectors/LiveSelectors/index.ts:3]: and rename field joined to isJoined

File: agoraStreaming/src/Screens/Home/style.tsx [hunk 5]
```diff
@@ -12,11 +12,13 @@ type HomeScreenStyles = {
   logout: ViewStyle;
   marker: ViewStyle;
   createContainer: ViewStyle;
+  headerContainer: ViewStyle;
   markerImage: ImageStyle;
 };
 
 const SIZE_MARKER = 30;
 const BORDER_SIZE = 4;
+const BACKGROUND_RADIUS = 35;
```
Comment (Mentos15) [inline agoraStreaming/src/Screens/Home/style.tsx:21]: rename

**Review by Mentos15 — APPROVED** [review]

**Review by koverchik — APPROVED** [review]

**Pull request merged**